README.md

```diff
@@ -16,7 +16,7 @@ If running Windows, download the latest [release](https://github.com/danrahn/Int
 In Docker:
 
 ```bash
-docker run -p 3233:3232 \
+docker run -p 3232:3232 \
```

Comment:
This isn't a typo, it's mapping port 3232 in the container to port 3233 on the host, as explained in the wiki: https://github.com/danrahn/IntroEditorForPlex/wiki/Configuration#docker